README.md

```diff
@@ -16,15 +16,15 @@ If you see a step below that could be improved (or is outdated), please update t
 
 - We recommend using [nvm](https://github.com/creationix/nvm): `nvm install && nvm use`.
 
-2. Make sure you have a PostgreSQL database available
+2. Make sure you have a PostgreSQL database available.
 
-- Check the version: 11.0, 10.3, 9.6.8, 9.5.12, 9.4.17, 9.3.22 or newer
+- Check the version (in production we're currently running 13.3)
 - Check that the [PostGIS](https://postgis.net/install/) extension is available
 - More info in our [PostgreSQL Database](docs/postgres.md) documentation
 
-3. For [node-gyp](https://github.com/nodejs/node-gyp), make sure you have Python 2 available and configured as the active version.
+3. For [node-gyp](https://github.com/nodejs/node-gyp), make sure you have Python 3 available and configured as the active version.
```

Comment:
I did not know this was compatible with Python 3, it used to be Python 2 only.

Reply:
It seems to be heavily dependent on on the version of `node-gyp`, I can look further into it and update this language